backend/typescript/utilities/stripeUtils.ts

```diff
@@ -90,6 +92,83 @@ export async function createStripePrice(
   return priceObject;
 }
 
+export const createStripeLineItems = (
+  sessionsToRegister: CampSession[],
+  campers: CreateCampersDTO,
+  dropoffPriceId: string,
+  pickupPriceId: string,
+): Stripe.Checkout.SessionCreateParams.LineItem[] => {
+  return sessionsToRegister.flatMap((campSession) => {
+    const priceItems = [
+      { price: campSession.campPriceId, quantity: campers.length },
+    ];
+
+    // every camper belongs to every session
+    // all same EDLP price
+    // campers have different EDLP dates, which may belong to different sessions
+    // we want to isolate EDLP per session
```

Comment:
todo remove this

Reply:
If you keep this logic, I think it's worth having explanation comments here

Reply:
yeah this will disappear soon